google-cloud-translate-v2/README.md

```diff
@@ -0,0 +1,74 @@
+# Ruby Client for the Cloud Translation V2 API
+
+API Client library for the Cloud Translation V2 API
+
+Cloud Translation can dynamically translate text between thousands of language pairs. Translation lets websites and programs programmatically integrate with the translation service.
+
+https://github.com/googleapis/google-cloud-ruby
+
+## Installation
+
+```
+$ gem install google-cloud-translate-v2
+```
+
+## Before You Begin
+
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+1. {file:AUTHENTICATION.md Set up authentication.}
+
+## Quick Start
+
+```ruby
+require "google/cloud/translate/v2"
+
+client = ::Google::Cloud::Translate::V2.new
```

Comment:
We don't usually use `::Google` scoping in examples. Is it needed here?

Reply:
Updated.